src/scss/_bases.scss

```diff
@@ -54,7 +60,7 @@ article {
 }
 
 body {
-    color: #73879C;
+    color: $default-light-blue;
     background: #2A3F54;
```

Comment:
@atais Yes, that's the idea. But a quick css-to-scss convert of the original custom.css file revealed that there are about 170 different color hexes and rgba's. So that's easier said than done.